phpcount.php

```
@@ -135,8 +132,8 @@ public static function GetHits($pageID, $unique = false)
         }
         else
         {
-            die("Missing hit count from database!");
-            return false;
+            //die("Missing hit count from database!");
+            return 0;
```

There's no reason this should be 0 instead of false, I think it should actually throw an exception, but I won't let that block this PR.

---

If, in the web page, you call GetHits _before_ AddHits now, the entry in the database may not yet exist (for a new page). If there is no entry, I found it fit to return 0, as a defensive programming measure (I stumbled upon this after manually deleting db entries while testing :-)

---

Oh, I see. Yes, what you did is correct!

---

Hmm, wait, how does this distinguish "the row simply isn't present in the database yet" from "the connection to the database catastrophically failed, so we don't know what the hit count is"? In the former case returning 0 is correct, but in the latter case it should throw an exception.